proof/crefine/AARCH64/SR_lemmas_C.thy

```diff
@@ -411,11 +411,15 @@ lemma cmdbnode_relation_mdb_node_to_H [simp]:
   unfolding cmdbnode_relation_def mdb_node_to_H_def mdb_node_lift_def cte_lift_def
   by (fastforce split: option.splits)
 
-definition
-  tcb_no_ctes_proj :: "tcb \<Rightarrow> Structures_H.thread_state \<times> machine_word \<times> machine_word \<times> arch_tcb \<times> bool \<times> word8 \<times> word8 \<times> word8 \<times> nat \<times> fault option \<times> machine_word option"
+definition tcb_no_ctes_proj ::
+  "tcb \<Rightarrow> Structures_H.thread_state \<times> machine_word \<times> machine_word \<times> arch_tcb \<times> bool \<times> word8
+   \<times> word8 \<times> word8 \<times> nat \<times> fault option \<times> machine_word option
```

Comment:
I think this indentation is incorrect. The product belongs entirely to the right argument, so the entire product should be to the right of `⇒ `